daemon/cmd/daemon_main.go

```diff
@@ -823,6 +823,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Duration(option.CTMapEntriesTimeoutSYNName, 60*time.Second, "Establishment timeout for entries in TCP CT table")
 	option.BindEnv(vp, option.CTMapEntriesTimeoutSYNName)
 
+	flags.Duration(option.CTMapEntriesTimeoutClosingName, 30*time.Second, "Closing Timeout for entries in TCP CT table")
```

Comment:
Why `30` seconds? I think the Linux kernel has `60` seconds of `TIME_WAIT` duration.

Reply:
Let me change it to 60 seconds as the default value then user (like us) can specify shorter time by the parameter.